tests/foreman/ui/test_registration.py

```diff
@@ -445,9 +450,10 @@ def test_global_re_registration_host_with_force_ignore_error_options(
 
 
 @pytest.mark.tier2
+@pytest.mark.rhel_ver_match('8')
 @pytest.mark.usefixtures('enable_capsule_for_registration')
```

Comment:
As you're not using fixture default_smart_proxy, can we remove use of this enable_capsule_for_registration fixture as well?

Reply:
Yeah. I was thinking to remove it.

Reply:
@shweta83 its removed for a few tests, but it is still used for many other tests in this module